webknossos-datastore/app/com/scalableminds/webknossos/datastore/storage/FileSystemsHolder.scala

```diff
@@ -22,24 +15,24 @@ object FileSystemsHolder extends LazyLogging {
   val schemeHttp: String = "http"
   val schemeGS: String = "gs"
 
-  private val fileSystemsCache: AlfuFoxCache[RemoteSourceDescriptor, FileSystem] =
+  private val vaultCache: AlfuFoxCache[RemoteSourceDescriptor, VaultPath] =
```

Comment:
I find it a bit weird to cache VaultPaths here, when semantically what we want to cache is the Vaults themselves. How do you feel about this?
Maybe it makes more sense to have DataVault.create return a DataVault, not a path. That can be cached here.

The public method here could still return a path. Maybe something like

```
getVaultPath(remoteSourceDescriptor: RemoteSourceDescriptor): Fox[VaultPath] =
for {
valut <- vaultCache.getOrLoad(remoteSourceDescriptor, create)
} yield (new VaultPath(remoteSourceDescriptor.uri, vault)
```
if that makes sense?